### PR TITLE
Generalised/tense-corrected the example

### PR DESCRIPTION
**Pull request by yashpalgoyal1304**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [x] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- Updated the tense of the example so that it aligns with the general case it's intended to point to. it helps to avoid misunderstanding about current distribution status.

#### Fixes the following issue(s)

- Fixes #

#### APK testing 

On the website the APK can be found by going to the "Checks" tab below the title and then on "artifacts" on the right.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by opusforlife2** [comment]
The text in brackets is a hypothetical example of something that might happen every now and then, not a reference to the one particular incident you're thinking of where Newpipe was stuck on an old version on F-Droid. So the current version is correct and there is no change needed.

But thanks for the thought, Yash. ;)

**Comment by yashpalgoyal1304** [comment]
`/close` ?

File: README.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ You can install NewPipe using one of the following methods:
 
 We recommend method 1 for most users. APKs installed using method 1 or 2 are compatible with each other, but not with those installed using method 3. This is due to the same signing key (ours) being used for 1 and 2, but a different signing key (F-Droid's) being used for 3. Building a debug APK using method 4 excludes a key entirely. Signing keys help ensure that a user isn't tricked into installing a malicious update to an app.
 
-In the meanwhile, if you want to switch sources for some reason (e.g. NewPipe's core functionality was broken and F-Droid doesn't have the update yet), we recommend following this procedure:
+In the meanwhile, if you want to switch sources for some reason (e.g. NewPipe's core functionality was broken and F-Droid didn't have the update for quite some weeks), we recommend following this procedure:
```
Comment (yashpalgoyal1304) [inline README.md:98]:
```suggestion
In the meanwhile, if you want to switch sources for some reason (e.g. say NewPipe's core functionality brakes and F-Droid doesn't have the update yet), we recommend following this procedure:
```

Comment (yashpalgoyal1304) [reply README.md:98]: how about this? that explicitly states the hypothetical-ity of the example.

Comment (mhmdanas) [reply README.md:98]: I would say that the "e.g." already expresses that it's just a hypothetical example. If you decide to change it though, it should be "breaks", not "brakes".

Comment (mhmdanas) [reply README.md:98]: Also, I think "yet" was fine; no need for "quite some weeks."

Comment (yashpalgoyal1304) [reply README.md:98]: yeah, i have made it to be "yet" in the new suggestion. i hope u checked it. :)

Comment (opusforlife2) [reply README.md:98]: "e.g. say" is redundant. Use only one of them. I think "e.g." will be more universally understood. Also "latest update".
```suggestion
In the meanwhile, if you want to switch sources for some reason (e.g. NewPipe's core functionality breaks and F-Droid doesn't have the latest update yet), we recommend following this procedure:
```

**Review by mhmdanas — APPROVED** [review]
LGTM. Not merging right now to know @opusforlife2's and possibly others' opinions.

**Comment by yashpalgoyal1304** [comment]
> LGTM ?

**Comment by mhmdanas** [comment]
It means "looks good to me."

**Comment by opusforlife2** [comment]
There's also the classic "LGTM to me!".

@yashpalgoyal1304 Please change the title and body, as this PR is now doing something completely different.

@mhmdanas It's asking me to approve and run the workflow. Is it okay to do that at any point or is there something to look out for?

Feel free to merge this after he changes the title and body.

**Comment by mhmdanas** [comment]
@opusforlife2 feel free to do it now; there isn't really anything to look for. The reason for this requirement is elaborated in https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks:

> Forks of public repositories can submit pull requests that propose changes to a repository's GitHub Actions workflows. Although workflows from forks do not have access to sensitive data such as secrets, they can be an annoyance for maintainers if they are modified for abusive purposes. To help prevent this, workflows on pull requests are not run automatically if they are received from first-time contributors, and must be approved first.

**Comment by opusforlife2** [comment]
@mhmdanas That seems like throwing the baby out with the bath water. Seems like it would be more helpful if first-time contributors were prevented from editing the workflow files only.

**Comment by mhmdanas** [comment]
Well, take that issue up with the GitHub developers, not me. :P

**Comment by yashpalgoyal1304** [comment]
> change t & b

done

**Comment by mhmdanas** [comment]
Thanks!

**Pull request merged**